easybuild/easyconfigs/c/CADD/CADD-v1.4.eb

```diff
@@ -0,0 +1,191 @@
+#
+# EasyConfig for Combined Annotation Dependent Depletion (CADD).
+#
+
+easyblock = 'Tarball'
+
+name = 'CADD'
+version = 'v1.4'
+
+homepage = 'http://cadd.gs.washington.edu/home'
+description = """
+Tool for scoring the deleteriousness of variants in the human genome(v1.4).
+"""
+
+#
+# Don't use 'dummy' as toolchain version to make sure dependencies are loaded
+# both at build as well as at runtime. With 'dummy' as toolchain version,
+# dependencies will only be loaded at runtime.
+#
+#toolchain = {'name': 'dummy', 'version': 'dummy'}
+toolchain = {'name': 'dummy', 'version': ''}
+
+dependencies = [
+    ('PythonPlus', '2.7.12', '-v18.11.1', ('foss', '2015b')),
+    ('HTSlib', '1.3.2', '', ('foss', '2015b')),
+    ('SAMtools', '1.5', '', ('foss', '2015b')),
+    ('VEP', '90.5', '', ('dummy', '')),
+]
+
+#
+# Example URL: 
+# http://krishna.gs.washington.edu/download/%(name)s/v1.3/cadd_v1.3_scripts.tgz
+#
+source_urls = [ 
+    # Only for annotations which is built on genome build GRCh37/hg19
+    'https://krishna.gs.washington.edu/download/%(name)s/%(version)s/GRCh37/'
+]
+
+sources = [
+	#
+	# Use manually downloaded CADD-scripts-master from github.
+	#
+    {
+        'filename': 'CADD-scripts-master.zip',  
```

Comment:
The file CADD-scripts-master.zip is not available from URL 'https://krishna.gs.washington.edu/download/%(name)s/%(version)s/GRCh37/'. It's an URL from GitHub which is missing from the list of source_urls...

The problem with a *master.zip from GitHub is that it is always the latest and greatest version of the master branch, so this is stable. You can also get a stable URL for what is currently the "master" by using the latest commit. For example this URL is for a commit from 10 days ago: https://github.com/kircherlab/CADD-scripts/archive/232fc0bda9f9ed127694f1d38669db043918328e.zip

Reply:
No, it's not avaiable from the official sit. For this, it's a little bit tricky. I `git clone` the master repository of CADD v1.4 and compressed it by zip. Then made a *fake* `md5` to help it pass the `md5` check. That's why it failed to download the zip file when you're testing but works when I test it in a user dir. Any better resolution for this issue?